state/runtime/evm/state.go

```diff
@@ -44,10 +44,10 @@ var (
 	errInvalidJump           = errors.New("invalid jump destination")
 	errOpCodeNotFound        = errors.New("opcode not found")
 	errReturnDataOutOfBounds = errors.New("return data out of bounds")
+	errInvalidGasValue       = errors.New("invalid gas value")
```

Comment:
```suggestion
	errInvalidBalance      = errors.New("invalid balance")
```

Or possibly it is even better approach to merge `errInvalidGasValue` and `errInvalidMessageValue` and provide the single error e.g. `errInvalidUint256Value = errors.New("invalid unit 256 value")` WDYT?

cc @goran-ethernal

Reply:
It depends on what information has higher value - do we need to pinpoint exactly invalid variable or we will implement error post processing on extracting this information in the EVM loop (afaik, GO does not propagate call stack with error). This approach ensures that when the error appears, its location is known without additional processing.

Reply:
In that case, `invalid gas value` is misleading, because it is not the `gas` parameter that is in question but the `balance`. That's why I was suggesting change the error message to `invalid balance value`.

Reply:
I see, changing it based on that.